auto_aim_node:
  ros__parameters:
    color_threashold: 13
    gray_threashold: 24
    DEBUG: True
    camere_matrix: [1293.5303221625442802, 0.3651215140945823, 355.9091806402759630, 
                    0.0000000000000000, 1293.9256252855957428, 259.1868664367483461,
                    0.0000000000000000, 0.0000000000000000, 1.0000000000000000]
    dist_coeffs: [-0.2126367859619807, 0.2282910064864265, 0.0020583387355406, 0.0006136511397638, -0.7559987171745171]